src/game/demo/mod.rs

```diff
@@ -1,9 +1,10 @@
+pub(crate) mod fx;
 pub(crate) mod level;
 
 use bevy::prelude::*;
 use bevy_auto_plugin::auto_plugin::*;
 
 #[auto_plugin(app=app)]
 pub(crate) fn plugin(app: &mut App) {
-    app.add_plugins(level::plugin);
+    app.add_plugins((fx::plugin, level::plugin));
```

Comment:
Nitpicking - I'm weird and like having one line per plugin just so it's easy to comment them out for testing when there's not enough to make it multi line

Reply:
Done